lib/assets.js

```diff
@@ -102,17 +110,27 @@ Assets.prototype.serveAsset = function (req, res, next) {
 
   var contentType = asset.contentType;
 
-  if (contentType.match(/^text\/|\/json$|\/javascript$/)) {
-    contentType += "; charset=UTF-8";
-  }
+  if (contentType) {
+    if (contentType.match(/^text\/|\/json$|\/javascript$/)) {
+      contentType += "; charset=UTF-8";
+    }
 
-  res.setHeader("Content-Type", contentType);
+    res.setHeader("Content-Type", contentType);
+  }
```

Comment:
Mh, that would mean we have to resolve the mime type.
I just pushed a new commit that required the `mime` module but then is able to resolve the mime type of the requested asset.

What do you think?